homeworks/07-mars-viewer/src/store/favourites/favouritesSlice.ts

```diff
@@ -0,0 +1,25 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { RoverName } from '../../types';
+
+type Favourite = { id: number; sol: number; rover: RoverName };
```

Comment:
since photos ids are unique, storing sol and rover may be redundant, so better store id only and find this photo by id only